src/docs/welcome/quickStart/DOC.md

```diff
@@ -89,31 +94,31 @@ Bootstrapped application with essential tools for rapid development
 1. Run the container with default options
 
     ```sh
-    bash scripts/run.sh
+    SPANDX_CONFIG=path/to/insights-frontend-starter-app/config/spandx.config.js sh path/to/insights-proxy/scripts/run.sh
```

Comment:
Hey this is not a valid path anymore right?
landing-page-frontend/proxy-config/local-frontend.js

Is the new path it seems.

Reply:
I'm not sure why the landing page uses that path, but in the starter app it's:
`config/spandx.config.js`

https://github.com/RedHatInsights/insights-frontend-starter-app/blob/master/config/spandx.config.js